packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterFormContainer.js

```diff
@@ -91,7 +93,23 @@ class RegisterFormContainer extends Component {
     return regex.test(errorMessage)
   }
 
+  async onChange (event, formErrorHandler) {
+    const { target: { name, value } } = event
+    if (name === 'username') {
+      this.checkNameForConflict(value)
+        .then((errorMessage) => {
+          if (errorMessage) formErrorHandler('username', errorMessage)
+        })
```

Comment:
Should this block be using `await`? Something like:
```
const errorMessage = await this.checkNameForConflict(value)
```

Reply:
Oh yeah, I think I started switching it to async/await and didn't finish. Thanks.